src/styles/text.tsx

```diff
@@ -22,11 +22,11 @@ export const H2 = styled.h2<{ $color?: string }>`
   margin: 0;
 `;
 
-export const H3 = styled.h3<{ $color?: string }>`
+export const H3 = styled.h3<{ $color?: string; $fontWeight?: number | string }>`
   ${openSans.style}
   font-size: 1.5rem;
-  font-weight: 600;
   color: ${props => props.$color || COLORS.greyDarker};
+  font-weight: ${props => props.$fontWeight || '600'};
```

Comment:
I believe we decided to use the practice of destructuring. Thus, here we would do `font-weight: ${({ $fontWeight }) => $fontWeight || '600'}` instead. The `color` prop is an artifact from a time long past, it would be great if you can update that to use destructuring as well 👍